templates/tags/list.html

```diff
@@ -26,7 +27,9 @@
                 {% else %}
                     <a href="{{ term.permalink | safe }}">
                         {{ term.name }}</a>
-                        – {{ term.pages | length }}{%- if term.pages | length == 1 %}
+                        &nbsp;– &nbsp;
```

Comment:
✅ `post_pre` translation is good
🛠️ there might be too many spaces and `&nbsp;`. Will need to review with final PR